clicommand/artifact_download.go

```diff
@@ -22,17 +22,22 @@ Description:
 
 Example:
 
-   $ buildkite-agent artifact download "pkg/*.tar.gz" . --build xxx
+   $ buildkite-agent artifact download --build xxx "pkg/*.tar.gz" .
 
    This will search across all the artifacts for the build with files that match that part.
    The first argument is the search query, and the second argument is the download destination.
 
    If you're trying to download a specific file, and there are multiple artifacts from different
    jobs, you can target the particular job you want to download the artifact from:
 
-   $ buildkite-agent artifact download "pkg/*.tar.gz" . --step "tests" --build xxx
+   $ buildkite-agent artifact download --step "tests" --build xxx "pkg/*.tar.gz" .
 
-   You can also use the step's jobs id (provided by the environment variable $BUILDKITE_JOB_ID)`
+   You can also use the step's jobs id (provided by the environment variable $BUILDKITE_JOB_ID)
+
+   By default, only artifacts from the most recent job in a chain of retried jobs are downloaded.
+   To include artifacts from previous retried jobs, use the "--include-retried-jobs" flag:
```

Comment:
I wonder if this makes it clearer? “Retried” really feels like it does double duty here:

```suggestion
   By default, only artifacts from the last job in a chain of retried jobs are downloaded.
   To include artifacts from superseded jobs, use the "--include-retried-jobs" flag:
```

Reply:
I'm less keen on superseded, because if I read that I'd be wondering if it meant something specific in this context. We could just use "from previous jobs"


Huh, that was unintentional:
![image](https://user-images.githubusercontent.com/95874/100632776-89994400-332d-11eb-8bd5-1fe119fb6470.png)

Reply:
I’m just not sure “previous jobs” conveys “previous tries of this step” though, it seems ambiguous whether this means “prior jobs in the build” or something else